mailchimp3/baseapi.py

```diff
@@ -42,16 +42,28 @@ def _iterate(self, url, **kwargs):
         kwargs['count'] = integer
         kwargs['offset'] = integer
         """
+        #fields as a kwarg ought to be a string with comma-separated substring
+        #values to pass along to self._mc_client._get(). it also ought to
+        #contain 'total_items' in all cases, so that is enforced here
+        if kwargs.get('fields') is None:
+            kwargs['fields'] = 'total_items'
```

Comment:
Adding this here will prevent any other data from being returned. If the fields queryparam is not passed, all fields are returned, if we pass just the total_items, we would essentially just be returning the count.